### PR TITLE
Prepares watchmaker 0.13.0 release

### PR DESCRIPTION
**Pull request by confusdcodr**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
 
+### 0.13.0
+
+**Commit Delta**: [Change from 0.13.0 release](https://github.com/plus3it/watchmaker/compare/0.12.1...0.13.0)
+
+**Released**: 2018.01.29
+
+**Summary**:
+
+*   add new salt formula amazon-inspector-formula
```
Comment (lorengordon) [inline CHANGELOG.md]: Can go ahead and summarize the change to the class structure, including a link to the PR. See the Changelog entry for [0.9.5](https://github.com/plus3it/watchmaker/blob/develop/CHANGELOG.md#095) as an example.

Also, here's the format we used last time we added a formula, might as well keep it consistent:

```
*   nessus-agent-formula
    -   New salt formula distributed with Watchmaker
```

Comment (confusdcodr) [reply CHANGELOG.md]: Done

**Comment by confusdcodr** [comment]
go codebuild go

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+
+**Commit Delta**: [Change from 0.13.0 release](https://github.com/plus3it/watchmaker/compare/0.12.1...0.13.0)
+
+**Released**: 2018.01.29
+
+**Summary**:
+
+*   amazon-inspector-formula 
+    - New salt formula distributed with watchmaker
+    - installs amazon-inspector formula
```
Comment (lorengordon) [inline CHANGELOG.md]: Gonna nit further: first letter capitalize, and the _formula_ installs the Amazon Inspector _agent_.

Comment (lorengordon) [inline CHANGELOG.md]: Also need a line item for `ash-linux-formula` that summarizes the changes there. See its commit log.

Comment (confusdcodr) [reply CHANGELOG.md]: Done

**Comment by confusdcodr** [comment]
go codebuild go

**Pull request merged**